nbs/examples/Forecasting_TFT.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>This could all be done before and already saved in the parquet file. Its 10 lines of very specific code that has not relation. at all with TFT. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done</p>